app/src/main/java/org/fossasia/susi/ai/skills/feedback/adapters/recycleradapters/AllReviewsAdapter.kt

```diff
@@ -40,7 +42,15 @@ class AllReviewsAdapter(val context: Context, val feedbackList: List<Feedback>?)
                 if (feedbackList[position].email != null &&
                         !TextUtils.isEmpty(feedbackList[position].email)) {
                     Utils.setAvatar(context, feedbackList.get(position).email, holder.avatar)
-                    Utils.truncateEmailAtEnd(feedbackList.get(position).email)?.let { holder.feedbackEmail?.text = it }
+                    if (PrefManager.getToken() != null) {
+                        if (!feedbackList.get(position).email.equals(PrefManager.getStringSet(Constant.SAVED_EMAIL).iterator().next().toString(), true)) {
```

Comment:
we shall used strings file here, instead of constants file.

Reply:
@codedsun This is fine because there is no duplication here. In the PR where you were asked to use stings, there we had duplicatIon i.e. same keys were being referenced from string resource files in some files while in others it was beig done from Constants file. That resulted in duplication, which is bad.